From 53d0693a89653e495f31fe93ffc8475ffd3baafc Mon Sep 17 00:00:00 2001
From: Libin Yang <libin.yang@intel.com>
Date: Tue, 4 Dec 2018 12:03:40 +0800
Subject: [PATCH 1079/1548] VBS: export vbs symbols

This patch exports some vbs symbols for other modules to use.

Tracked-On: PKT-1592
Tracked-On: projectacrn/acrn-hypervisor#1915
Signed-off-by: Libin Yang <libin.yang@intel.com>
Reviewed-by: Zhao Yakui <yakui.zhao@intel.com>
---
 drivers/vbs/vq.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/drivers/vbs/vq.c b/drivers/vbs/vq.c
index 9f7a829c7a67..4ea44c47b871 100644
--- a/drivers/vbs/vq.c
+++ b/drivers/vbs/vq.c
@@ -243,6 +243,7 @@ int virtio_vq_getchain(struct virtio_vq_info *vq, uint16_t *pidx,
 	       name, i);
 	return -1;
 }
+EXPORT_SYMBOL(virtio_vq_getchain);
 
 /*
  * Return the currently-first request chain back to the available queue.
@@ -254,6 +255,7 @@ void virtio_vq_retchain(struct virtio_vq_info *vq)
 {
 	vq->last_avail--;
 }
+EXPORT_SYMBOL(virtio_vq_retchain);
 
 /*
  * Return specified request chain to the guest, setting its I/O length
@@ -288,6 +290,7 @@ void virtio_vq_relchain(struct virtio_vq_info *vq, uint16_t idx,
 	vue->len = iolen;
 	vuh->idx = uidx;
 }
+EXPORT_SYMBOL(virtio_vq_relchain);
 
 /*
  * Driver has finished processing "available" chains and calling
@@ -340,6 +343,7 @@ void virtio_vq_endchains(struct virtio_vq_info *vq, int used_all_avail)
 	if (intr)
 		virtio_vq_interrupt(dev, vq);
 }
+EXPORT_SYMBOL(virtio_vq_endchains);
 
 /*
  * Initialize the currently-selected virtqueue.
-- 
2.20.1

